moveit_setup_assistant/templates/moveit_config_pkg_template/package.xml.template

```diff
@@ -20,6 +20,7 @@
   <run_depend>joint_state_publisher</run_depend>
   <run_depend>robot_state_publisher</run_depend>
   <run_depend>xacro</run_depend>
+  <run_depend>warehouse_ros_mongo</run_depend>
```

Comment:
that might be a problem? depends on a package that's outside of the moveit repo?